include/sound/compress_offload.h

```diff
@@ -21,7 +21,7 @@
 #include <linux/types.h>
 #include <sound/asound.h>
 #include <sound/compress_params.h>
-#define SNDRV_COMPRESS_VERSION SNDRV_PROTOCOL_VERSION(0, 2, 0)
+#define SNDRV_COMPRESS_VERSION SNDRV_PROTOCOL_VERSION(0, 3, 0)
```

Comment:
heh! GH doesnt let comment on patchlog...
I guess it would make sense to say, move the header to version 0.3.0...